examples/ansible/setup_neuvector.yml

```diff
@@ -0,0 +1,109 @@
+---
+# This Ansible playbook is used to manage the NeuVector workload on a ALP host.
+# The playbook supports two operations: setup and removal of NeuVector.
+# The operation is determined by the 'neuvector_state' variable, which can be set to 'present' or 'absent'.
+# When 'neuvector_state' is set to 'present', the playbook will:
+#   - Check if the tools for the NeuVector workload are already installed
+#   - Set SELinux into permissive mode
+#   - Retrieve the NeuVector image
+#   - Install the necessary tools for the NeuVector workload
+#   - Start the NeuVector service
+#   - Enable the NeuVector service to start on a reboot
+#   - Print a message with the connection details for NeuVector
+# When 'neuvector_state' is set to 'absent', the playbook will:
+#   - Stop the NeuVector service
+#   - Uninstall the tools for the NeuVector workload
+#   - Remove the NeuVector image
+
+- name: Running the NeuVector workload
+  hosts: alphost
+  vars:
+    neuvector_state: present
+    workload:
+      name: neuvector
+      image: registry.opensuse.org/suse/alp/workloads/bci_containerfiles/suse/alp/workloads/neuvector-demo:latest
+
+  tasks:
+    - name: Setup NeuVector
+      when:
+        - neuvector_state == "present"
+      block:
+        - name: Check if tools for workload {{ workload.name }} are already installed
+          become: true
+          ansible.builtin.command: podman container inspect {{ workload.name }}-allinone
```

Comment:
Running podman via command/shell is not great Ansible, given that there are containers.podman.xxx actions available in `containers.podman` collection that is part of the Ansible standard collections in Ansible 7.x and 8.1.

See [alp-test-env/ansible/roles/alp_workload/tasks/get_image.yml](https://github.com/SUSE/alp-test-env/blob/main/ansible/roles/alp_workload/tasks/get_image.yml) for an example of where I used them to interrogate image details and retrieve images in the [SUSE/alp-test-env][https://github.com/SUSE/alp-test-env/] that was created to test & validate this project.

Reply:
Now using containers.podman.podman_container_info(https://docs.ansible.com/ansible/latest/collections/containers/podman/podman_container_info_module.html)  updated the when condition accordingly.